### PR TITLE
reorg: opend3d::gui -> open3d::visualization::gui

### PR DESCRIPTION
**Pull request by pvarvak**
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/1979)

### CODE REVIEW

**Comment by germanros1987** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 14
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/intel-isl/Open3D/pullRequest?prid=5729245&bid=18743118)

File: reorg/mvdirs.bash [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#source from /bin/bash
```
Comment (yxlao) [inline reorg/mvdirs.bash:1]: remove this folder

Comment (pvarvak) [reply reorg/mvdirs.bash]: done

**Comment by yxlao** [comment]
after fixing the style check, it is good to merge

**Pull request merged**